Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -58,7 +58,11 @@ public float sprintMoveSpeed
 		get { return _sprintMoveSpeed; }
 	}
 
-	float _jumpMoveSpeed = 0f;
+	[ReadOnly("Jump move speed")]
+	public float jumpMoveSpeed = 0f;
```

Comment:
Nevermind, this value is set externally. I think we should instead look at a better way of managing the move speed when jumping. Rather than having a fixed move speed like that's set externally, we should have a max move speed (which would just be the sprinting speed), and just cap the move speed if the player keeps trying to move on one direction.